CONTRIBUTING.md

```diff
@@ -41,6 +61,9 @@ to be installed when running rustfmt. You can install the nightly version of Rus
 rustup install nightly
 ```
 
+**Important**: Do not modify formatting configuration (`rustfmt.toml`, `.clippy.toml`) without
+prior discussion. These changes affect all contributors and should be carefully considered.
```

Comment:
```suggestion
```suggestion
> [!IMPORTANT]  
> Do not modify formatting configuration (`rustfmt.toml`, `.clippy.toml`) without
prior discussion. These changes affect all contributors and should be carefully considered.
```
```